CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 - OE SDK Release packages can now be built on non-SGX and non-FLC machines.
+- Support for arbitrarily large thread-local data.
```

Comment:
Shouldn't this line say "for SGX machines"?   I don't see any changes for anything other than SGX.

Reply:
changed as suggested